webextension/popup.js

```diff
@@ -583,10 +586,14 @@ function startCheckMaybeWithWarning(tabs) {
                 preferredVariants.push(items.caVariant);
             }
             if (items.allowRemoteCheck === true) {
-                doCheck(tabs, "manually_triggered");
-                const newCounter = items.usageCounter + 1;
-                Tools.getStorage().set({'usageCounter': newCounter}, function() {});
-                chrome.runtime.setUninstallURL("https://languagetool.org/webextension/uninstall.php");
+                if (tabs.length > 0) {
+                    doCheck(tabs, "manually_triggered");
+                    const newCounter = items.usageCounter + 1;
+                    Tools.getStorage().set({'usageCounter': newCounter}, function() {});
+                    if (chrome.runtime.setUninstallURL) {
```

Comment:
Any special reason you added that `if`? Did `setUninstallURL` make problems?

Reply:
When we open popup.html on new the popup on FF, this function will be not exist.